jobs/JRRFS_UPP

```diff
@@ -26,11 +26,15 @@ export HISTORY_INTERVAL=${HISTORY_INTERVAL:-1}
 #-----------------------------------------------------------------------
 #
 export pid=${pid:-$$}
+export RUN="rrfs"${WGF}
 if [[ -z "${ENS_INDEX}" ]]; then
-  export RUN="rrfs"
+  export MEMID=""
+  export MEMDIR=""
 else
-  export RUN="ensrrfs"
+  export MEMID="_m${ENS_INDEX}"
+  export MEMDIR="/m${ENS_INDEX}"
 fi
+
 export jobid=${jobid:-${RUN}_upp_g${GROUP_INDEX}_${cyc}}
```

Comment:
can you please change this to have "MEMID" information, similar to other scripts? 
export jobid=${jobid:-${RUN}_upp${MEMID}_g${GROUP_INDEX}_${cyc}}

Reply:
Done